### DISCUSSION TITLE
Swipe to reply

### DISCUSSION THREAD

**Discussion by du82** [status-im/status-legacy#16651] [General]
Currently there's no ability to swipe to reply to messages, and its very slow to wait for the long press, then the drawer popup animation.

I propose swiping right to left to reply, that way it doesn't interfere with swiping left to right to go back.

**Comment by ibrkhalil** [comment]
Should haptics (Vibration) get triggered when swiping to reply?
And the swiping direction would be the opposite for RTL direction?
CC: @pedro-et @mariocnovoa

**Comment by cammellos** [comment]
Talked to @pedro-et, waiting for his reply, moving to discussion
